src/java/picard/sam/ValidateSamFile.java

```diff
@@ -58,20 +58,24 @@
 )
 public class ValidateSamFile extends CommandLineProgram {
     static final String USAGE_SUMMARY = "Validates a SAM or BAM file.  ";
-    static final String USAGE_DETAILS = "This tool reports on the validity of a SAM or BAM file relative to the SAM format specification " +
-            "(see http://samtools.github.io/hts-specs/SAMv1.pdf), which is useful for troubleshooting errors encountered with other tools " +
```

Comment:
You are correct.  This got "lost" during a git merge.  I will add it back.  Thanks for catching it.